packages/cardano-services/test/PgBoss/stakePoolMetricsHandler.test.ts

```diff
@@ -0,0 +1,90 @@
+import { Cardano, StakePoolProvider } from '@cardano-sdk/core';
+import { CurrentPoolMetricsEntity } from '@cardano-sdk/projection-typeorm';
+import { DataSource } from 'typeorm';
+import { initHandlerTest, poolId } from './util';
+import { logger } from '@cardano-sdk/util-dev';
+import { savePoolMetrics } from '../../src/PgBoss';
+
+describe('stakePoolMetricsHandler', () => {
+  let dataSource: DataSource;
+
+  beforeAll(async () => {
+    const testData = await initHandlerTest();
+
+    ({ dataSource } = testData);
+  });
+
+  describe('savePoolMetrics', () => {
+    const partialExpectedRecord = {
+      activeSize: '0.0005',
+      activeStake: 42_000_000n,
+      apy: 0,
+      livePledge: 23_000_000n,
+      liveSaturation: '0.002',
```

Comment:
Floats returned as string don't match the entity type. Should use a transformer: https://stackoverflow.com/a/70127622

Reply:
c934ab9ab should solve this.
Part of the change is in 2a53e7b41 too